### DISCUSSION TITLE
SplitHTTP 有获得 REALITY 支持的可能吗？

### DISCUSSION THREAD

**Discussion by KobeArthurScofield** [XTLS/Xray-core#3788] [General]
目前 REALITY 支持的传输包括 TCP、HTTP/2 和 gRPC （不包含已经被移除的 DomainSocket）

目前使用的配置包含了 REALITY+TCP+Vision 和 REALITY+HTTP/2 （gRPC 相对 HTTP/2 还是重了一点），后面想试试 SplitHTTP

SplitHTTP 以后有没有可能也能配合 REALITY，还是到 REALITY 支持 QUIC/HTTP/3 才会有考虑的可能？

**Comment by mmmray** [comment]
Hello, I made a PR for this at #3816.

**Reply by KobeArthurScofield** [reply]
Oh my so fast
